wdl-lint/src/lib.rs

```diff
@@ -129,25 +138,46 @@ pub fn rules() -> Vec<Box<dyn Rule>> {
         Box::<rules::RedundantInputAssignment>::default(),
     ];
 
-    // Ensure all the rule ids are unique and pascal case
+    // Ensure all the rule ids are unique and pascal case, and related rules are
+    // valid and exist
```

Comment:
Can we also add a check that related rules aren't self-referential? e.g. rule `FooBar` should cause an error if `FooBar` appears in its related rules list.